packages/core/test/insured-bridge/BridgePool.js

```diff
@@ -432,11 +434,15 @@ describe("BridgePool", () => {
         return (
           ev.depositId.toString() === depositData.depositId.toString() &&
           ev.sender === depositData.l2Sender &&
+          ev.slowRelayer === relayer &&
           ev.depositTimestamp === depositData.depositTimestamp &&
           ev.recipient === depositData.recipient &&
           ev.l1Token === depositData.l1Token &&
           ev.amount === depositData.amount &&
-          ev.maxFeePct === depositData.maxFeePct &&
+          ev.slowRelayFeePct === depositData.slowRelayFeePct &&
```

Comment:
previously we were testing `maxFeePct`! this is null both within the event and locally (it was a previous implementation that has been refactored away). I changed it to correctly validate this even behavour.